torch/nn/functional.py

```diff
@@ -2523,9 +2523,9 @@ def _output_size(dim):
 
         # make scale_factor a tensor in tracing so constant doesn't get baked in
         if torch._C._get_tracing_state():
-            return [(torch.floor(input.size(i + 2) * torch.tensor(float(scale_factors[i])))) for i in range(dim)]
+            return [(torch.floor((input.size(i + 2) * torch.tensor(float(scale_factors[i]))).float())) for i in range(dim)]
```

Comment:
seems the old code should generate the float tensor as well, is .float() conversion necessary?

Reply:
Without this, the generated ONNX graph fails in ORT and complains that the input of the "Floor" node is an Int. 
I could submit an example of the ONNX file and the error if required.

For the following line, I did the same for consistency and out of precaution.

Reply:
Yea, i am wondering how the onnx model looke like? could you attach the exported model?